cli/replicaset/discovery.go

```diff
@@ -1,69 +1,39 @@
 package replicaset
 
-import (
-	"fmt"
-
-	"github.com/tarantool/tt/cli/connector"
-	"github.com/tarantool/tt/cli/running"
-)
-
 // Discoverer is an interface for discovering information about
 // replicasets.
 type Discoverer interface {
 	// Discovery returns replicasets information or an error.
-	Discovery() (Replicasets, error)
+	Discovery(force bool) (Replicasets, error)
```

Comment:
Boolean parameters are harder to maintain. If I see `discoverer.Discovery(true)` somewhere, I have no idea what the `true` does. Let's use an enum or split the function into `Discovery` and `ForcedDiscovery`, whatever that means.